shenyu-admin/src/main/java/org/apache/shenyu/admin/utils/CommonUpstreamUtils.java

```diff
@@ -65,7 +66,7 @@ public static DivideUpstream buildDivideUpstream(final String protocol, final St
     public static DubboUpstream buildDefaultDubboUpstream(final String host, final Integer port) {
         return DubboUpstream.builder().upstreamHost("localhost").protocol("dubbo://").upstreamUrl(buildUrl(host, port)).weight(50).warmup(10).timestamp(System.currentTimeMillis()).build();
     }
-    
+
```

Comment:
why not modify this

Reply:
you can format this code，

Reply:
E, The Idea shortcuts default formated . please Ingore those default formated, merge  convertCommonUpstreamList()  method That's enough .

Reply:
E, The Idea shortcuts default formated . please Ingore those default formated, merge  convertCommonUpstreamList()  method That's enough .

Reply:
hi， pls only modify convertCommonUpstreamList， not modify other~

Reply:
Ok, I revert  unnecessary formated, please check.